Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/DeploymentSDK/VisualStudioRemoteDeployer.exe.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup useLegacyV2RuntimeActivationPolicy="true"> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />

```

Comment:
addressed in commit [bae6112](https://github.com/mmikkelson/vsts-rm-extensions/commit/bae6112afe5e49f473b944a6fe99992ccbf0df97)